samples/boards/nrf52/mesh/onoff_level_lighting_vnd_app/src/mesh/device_composition.c

```diff
@@ -813,40 +664,58 @@ static void light_lightness_set_unack(struct bt_mesh_model *model,
 				      struct bt_mesh_msg_ctx *ctx,
 				      struct net_buf_simple *buf)
 {
-	u8_t tid;
+	u8_t tid, buf_len, delay;
 	u16_t actual;
+	s64_t now;
 	struct net_buf_simple *msg = model->pub->msg;
 	struct light_lightness_state *state = model->user_data;
 
+	buf_len = buf->len;
 	actual = net_buf_simple_pull_le16(buf);
 	tid = net_buf_simple_pull_u8(buf);
 
-	if (state->last_tid == tid && state->last_tx_addr == ctx->addr) {
+	now = k_uptime_get();
+	if (state->last_tid == tid && state->last_tx_addr == ctx->addr &&
+	    (now - state->last_msg_timestamp <= 6000)) {
```

Comment:
Perhaps K_SECONDS(6) instead of 6000? Same for the other similar place (which I wont comment on separately)

Reply:
It will add additional burden in calling a function.

Reply:
What makes you think that? K_SECONDS() is a macro that will evaluate to a build-time constant. Hint: macros are always capital case in Zephyr.

Reply:
I will do it shortly.

Reply:
I've to create separate commit since could not fixed commit 5 with commit 2 using rebase -i.